### ISSUE TITLE
Open by default

### ISSUE THREAD

**Issue by NickRaleigh** [Alhadis/Accordion#22]
Hi there,
Is there a way to have the accordion open by default? On mobile, I don't want all of the nested items hidden away on page load for my website.

**Comment by Alhadis** [comment]
Yes. Simply adding the `open` class to individual fold elements (before the accordion is initialised) should work:

~~~diff
 <ul class="accordion">
-	<li class="open">
+	<li class="open">
~~~
